charts/stacks/observability/values.yaml

```diff
@@ -145,7 +153,7 @@ thanos:
       logLevel: debug
       mode: dual-mode
       nodeSelector:
-        node.kubernetes.io/instance-type: r5.16xlarge
+        node.kubernetes.io/instance-type: r6i.16xlarge
       tsdbRetention: 14d
       resources:
         requests:
```

Comment:
@rsevilla87 we might also need to change line 178, also thoughts on setting nodeselector for other apps like grafana? if they get deployed on the same node as thanos and if thanos takes the node down, grafana stops working too?

Reply:
Hey @Harshith-umesh ; I only set nodeSelector for resource intensive applications such as elastic and Thanos. Grafana resource usage is very low. On the other hand, if a node goes down, grafana will be evicted and then re-scheduled to a different worker node, as it doesn't have any kind of nodeSelector.

Reply:
ack, i was just worried about the downtime before it gets redeployed on another node.